Sources/Articles/Documentation.docc/2025/jwt-kit/jwt-kit.md

```diff
@@ -0,0 +1,198 @@
+# Getting Started with Vapor's JWTKit 5: A Practical Introduction to JSON Web Tokens in Swift
+
+JWTKit is a library for working with JSON Web Tokens (JWT) in Swift. It provides a simple and easy-to-use API for creating, parsing, and validating JWTs. JWTs are a popular way to securely transmit information between parties, and JWTKit makes it easy to work with them in your Swift projects.
+
+In this tutorial, you'll learn how to use JWTKit to create and validate JWTs in Swift. You'll start by installing JWTKit using Swift Package Manager, then you'll learn how to create a JWT and validate it using JWTKit's API. At the end of this tutorial, you'll be able to implement an authentication and authorisation flow in your Swift projects using JWTs.
+
+## What is a JSON Web Token (JWT)?
+
+A JWT is a compact, URL-safe way to represent claims between two parties. The token is digitally signed, which makes it secure and trustworthy. JWTs are commonly used for authentication and authorisation in web applications, APIs, and microservices.
```

Comment:
```suggestion
A JWT is a compact, URL-safe way to represent claims between two parties. The token is digitally signed, which makes it secure and trustworthy. JWTs are commonly used for authentication and authorisation in (web) applications, APIs, and microservices.
```

Reply:
Would like to make it clear this is common in iOS apps too (probably a large chunk of our audience)